projects/Notepad++/raylib_npp_parser/raylib_to_parse.h

```diff
@@ -339,6 +339,7 @@ RLAPI void ImageAlphaClear(Image *image, Color color, float threshold);
 RLAPI void ImageAlphaMask(Image *image, Image alphaMask);                                                // Apply alpha mask to image
 RLAPI void ImageAlphaPremultiply(Image *image);                                                          // Premultiply alpha channel
 RLAPI void ImageBlurGaussian(Image *image, int blurSize);                                                // Apply Gaussian blur using a box blur approximation
+RLAPI void ImageKernelConvolution(Image *image, float* karnel, int karenlWidth);                         // Apply Custom Square image convolution kernel
```

Comment:
```suggestion
RLAPI void ImageKernelConvolution(Image *image, float* kernel, int kernelWidth);                         // Apply Custom Square image convolution kernel
```
Same with every other occurrence.

Reply:
Yeap, that's right